webapp/API/types/users-types.ts

```diff
@@ -13,6 +13,10 @@ export interface GetUserByFirebaseAuthIdDTO {
 	data: UserDTO;
 }
 
+export interface UpdateUserByIdDTO {
+	data: UserDTO;
+}
+
 // ----- type guards -----
```

Comment:
you need a type guard for the interface UpdateUserByIdDTO

Reply:
soga~